.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 'stable'
+install: npm install
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database barefoot_test;' -U postgres
+script: npm test
```

Comment:
Good job @WillyBoris for the progress.
As a suggestion for seamless work, we can add the configurations that will be used to trigger coveralls by the time we are working on coverage:
```
 after_success:
  - npm run coveralls
```

Reply:
yeah, it's very needed `npm run coverage`. Thanks